src/main/java/com/google/devtools/build/lib/rules/cpp/CcToolchainAliasRule.java

```diff
@@ -72,7 +76,14 @@ public ConfiguredTarget create(RuleContext ruleContext)
         throws InterruptedException, RuleErrorException, ActionConflictException {
 
       CcToolchainProvider ccToolchainProvider =
-          CppHelper.getToolchainUsingDefaultCcToolchainAttribute(ruleContext);
+          CppHelper.getToolchainUsingDefaultCcToolchainAttribute(ruleContext,
```

Comment:
Let me know if you prefer not to change the Java version of `cc_toolchain_alias` now that it has been Starlarkified and I will delete this part of the change.

Reply:
Yeah let's not change the Java version of cc_toolchain_alias. We should just delete the code. @buildbreaker2021 can the native code be removed?

Reply:
Ah never mind, this is the Rule definition which is needed for documentation for now. You can revert the change here though.

Reply:
@buildbreaker2021 we can keep the definition but delete the create() method.

Reply:
Yes, deleting this kinda slipped off my mind, will do it in several days

Reply:
Reverted the functional change, kept the attribute definition.

Reply:
I will do some more tests, I'm actually a bit confused this works with that part of the code removed.

Reply:
Done, this is a much smaller change now.